grpc-example/grpc-example-scala/hello-impl/src/test/scala/com/example/hello/impl/HelloServiceNoSslSpec.scala

```diff
@@ -1,60 +0,0 @@
-package com.example.hello.impl
-
-import akka.actor.ActorSystem
-import akka.grpc.GrpcClientSettings
-import com.example.hello.api._
-import com.lightbend.lagom.scaladsl.server.LocalServiceLocator
-import com.lightbend.lagom.scaladsl.testkit.ServiceTest
-import example.myapp.helloworld.grpc.{GreeterServiceClient, HelloRequest}
-import org.scalatest.BeforeAndAfterAll
-import org.scalatest.matchers.should.Matchers
-import org.scalatest.wordspec.AsyncWordSpec
-
-/**
- * Illustrate deliberate use of the gRPC client without TLS.
- */
-class HelloServiceNoSslSpec extends AsyncWordSpec with Matchers with BeforeAndAfterAll {
```

Comment:
This tests was doing exactly the same as HelloServiceSpec, but then without ssl. It's now obsolete.